src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/getCrumbDetails.tsx

```diff
@@ -42,7 +42,7 @@ function getCrumbDetails(type: BreadcrumbType) {
 
     case BreadcrumbType.INFO:
       return {
-        color: 'blue400',
+        color: 'blue300',
```

Comment:
Hmm maybe to differentiate from `QUERY`

```suggestion
        color: 'blue200',
```